tlovett1/safe-redirect-manager
# Quick Edit and Bulk edit

### Description of the Change
Added option to Quick edit and Bulk edit redirect's HTTPS status and Force Https meta on the post list table. 

Closes #98 

![Screenshot 2023-09-08 at 4 05 20 PM](https://github.com/10up/safe-redirect-manager/assets/6737105/5c8b636c-82c2-488e-8d0d-5386367c4068)
![Screenshot 2023-09-08 at 4 05 29 PM](https://github.com/10up/safe-redirect-manager/assets/6737105/eab3c2cd-7cb0-4ca7-a6b2-8db8b86cd3fc)


### How to test the Change

1. Goto Safe Redirect Manager's list page
2. Hover over any redirect and confirm the "Quick Edit" option available
3. On the Quick Edit, confirm the Https status code and Force Https meta fields available to edit
4. Confirm the updates are saved to the post
5. Select multiple posts and select the "Edit" option from the bulk actions dropdown
6. Confirm the data updated to respective posts



### Changelog Entry
> Added - Option to quick edit and bulk edit redirect's https status and force https meta.


### Credits
Props @dhanendran 


### Checklist:
